modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/XMLLog4jLoggersCheck.java

```diff
@@ -93,24 +96,29 @@ private void _checkLoggers(String fileName, String content)
 		}
 	}
 
-	private Set<String> _getSrcPaths() throws Exception {
-		if (!_srcPaths.isEmpty()) {
+	private synchronized List<String> _getSrcPaths() throws Exception {
+		if (_srcPaths != null) {
 			return _srcPaths;
 		}
 
+		_srcPaths = new ArrayList<>();
+
 		File file = getPortalDir();
 
 		List<String> fileNames = getFileNames(
 			file.getAbsolutePath(), new String[0],
 			new String[] {"**/com/liferay/**/*.java"});
 
 		for (String fileName : fileNames) {
+			fileName = StringUtil.replace(
+				fileName, CharPool.BACK_SLASH, CharPool.SLASH);
```

Comment:
@qz1010 Are you using `CharPool.BACK_SLASH` to catch Windows-style paths? If so, then we would need to check for __both__ Windows-style paths and Unix-style paths.

Reply:
Hi @qz1010, I misread this. This should be fine and should allow us to handle Windows-style paths.